`nepochs` field should be `n_epochs` in fine-tune request

https://platform.openai.com/docs/api-reference/fine-tunes/create#fine-tunes/create-n_epochs
```
com.theokanning.openai.OpenAiHttpException: Additional properties are not allowed ('nepochs' was unexpected)
        at com.theokanning.openai.service.OpenAiService.execute(OpenAiService.java:220)
        at com.theokanning.openai.service.OpenAiService.createFineTune(OpenAiService.java:119)
        at ... my code that calls createFineTune
        at java.base/java.lang.Thread.run(Thread.java:833)
 Caused by: retrofit2.adapter.rxjava2.HttpException: HTTP 400 
        at retrofit2.adapter.rxjava2.BodyObservable$BodyObserver.onNext(BodyObservable.java:57)
        at retrofit2.adapter.rxjava2.BodyObservable$BodyObserver.onNext(BodyObservable.java:38)
        at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:48)
        at io.reactivex.Observable.subscribe(Observable.java:10151)
        at retrofit2.adapter.rxjava2.BodyObservable.subscribeActual(BodyObservable.java:35)
        at io.reactivex.Observable.subscribe(Observable.java:10151)
        at io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual(ObservableSingleSingle.java:35)
        at io.reactivex.Single.subscribe(Single.java:2517)
        at io.reactivex.Single.blockingGet(Single.java:2001)
        at com.theokanning.openai.service.OpenAiService.execute(OpenAiService.java:211)
        ... 9 more
```

--- a/api/src/main/java/com/theokanning/openai/finetune/FineTuneRequest.java
+++ b/api/src/main/java/com/theokanning/openai/finetune/FineTuneRequest.java
@@ -1,5 +1,6 @@
 package com.theokanning.openai.finetune;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.*;
 
@@ -37,6 +38,7 @@ public class FineTuneRequest {
     /**
      * The number of epochs to train the model for. An epoch refers to one full cycle through the training dataset.
      */
+    @JsonProperty("n_epochs")
     Integer nEpochs;
 
     /**

--- a/api/src/main/java/com/theokanning/openai/finetune/FineTuneRequest.java
+++ b/api/src/main/java/com/theokanning/openai/finetune/FineTuneRequest.java
@@ -1,6 +1,5 @@
 package com.theokanning.openai.finetune;
 
-import com.fasterxml.jackson.annotation.JsonAlias;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.*;
 

--- a/client/src/test/java/com/theokanning/openai/FineTuneTest.java
+++ b/client/src/test/java/com/theokanning/openai/FineTuneTest.java
@@ -38,6 +38,7 @@ public class FineTuneTest {
         FineTuneRequest request = FineTuneRequest.builder()
                 .trainingFile(fileId)
                 .model("ada")
+                .nEpochs(4)
                 .build();
 
         FineTuneResult fineTune = service.createFineTune(request);

--- a/service/src/test/java/com/theokanning/openai/service/FineTuneTest.java
+++ b/service/src/test/java/com/theokanning/openai/service/FineTuneTest.java
@@ -38,6 +38,7 @@ public class FineTuneTest {
         FineTuneRequest request = FineTuneRequest.builder()
                 .trainingFile(fileId)
                 .model("ada")
+                .nEpochs(4)
                 .build();
 
         FineTuneResult fineTune = service.createFineTune(request);

